src/main/java/com/exadel/discountwebapp/discount/controller/DiscountController.java

```diff
@@ -17,8 +21,10 @@
     private final DiscountService discountService;
 
     @GetMapping
-    public List<DiscountResponseVO> findAll() {
-        return discountService.findAll();
+    public List<DiscountResponseVO> findAll(@RequestParam(value = "query", defaultValue = "", required = false) String query, Pageable pageable) {
```

Comment:
It seems like we should return Page object here to provide additional information about the page to UI. It should be the default behavior. In case when the page info is not passed in the request -> use default pageable instead.
Will discuss on the daily meeting

Reply:
done